src/util-internal.h

```diff
@@ -34,7 +34,7 @@ wifi_frequency_24;
 extern GHashTable*
 wifi_frequency_5;
 
-NETPLAN_ABI void
+NETPLAN_ABI void // XXX: still used in generate.c
```

Comment:
Do we need to keep the `NETPLAN_ABI` macro? We could replace it with `NETPLAN_INTERNAL`...

Reply:
Very true!